code/__DEFINES/dna_infuser_defines.dm

```diff
@@ -0,0 +1,17 @@
+
+/// just clarifying that no threshold does some special stuff, since only meme mutants have it
+#define DNA_INFUSION_NO_THRESHOLD ""
```

Comment:
This is a very odd define, are you sure we need this?

Reply:
The tsx presents the infuser_entry differently if there is no description on the treshold, which is why it's there.

Reply:
I just think any alternative would be better but it doesn't matter really.

Reply:
gomenasai